script/articleView.js

```diff
@@ -0,0 +1,21 @@
+var articleView = {};
+
+articleView.index = function() {
+  $('#aboutContent').hide();
+  var _renderAll = function() {
+    $articles = $('#articles');
+    $articles.show().siblings().hide();
+    Article.all.forEach(function(article) {
```

Comment:
This causes `Uncaught TypeError: Cannot read property 'forEach' of undefined` when I run your app.